.github/workflows/reusable-build-test-config.yml

```
@@ -79,7 +101,11 @@ jobs:
       - name: Prepare runner
         uses: XRPLF/actions/prepare-runner@2ece4ec6ab7de266859a6f053571425b2bd684b6
         with:
-          disable_ccache: false
+          disable_ccache: ${{ !inputs.ccache_enabled }}
+
+      - name: Set ccache log file
+        if: ${{ inputs.ccache_enabled && runner.debug }}
```

```suggestion
        if: ${{ inputs.ccache_enabled && runner.debug == '1' }}
```

Let's bee consistent with what's written next

---

Interesting, I thought it was a boolean variable, but the documentation indeed shows it is a string that always has the value '1' when enabled.